go/pkg/db/queries/key_insert.sql

```diff
@@ -14,6 +14,8 @@ INSERT INTO `keys` (
     created_at_m,
     enabled,
     remaining_requests,
+    refill_day,
```

Comment:
I took those out by design, cause they're no longer used and already in a different table. I guess we haven't deleted the columns yet